Profile/Profile/Presentation/Profile/ProfileViewModel.swift

```diff
@@ -118,9 +118,17 @@ public class ProfileViewModel: ObservableObject {
     
     @MainActor
     func logOut() async {
-        try? await interactor.logOut()
-        router.showLoginScreen()
-        analytics.userLogout(force: false)
+        do {
+            try await interactor.logOut()
+            router.showStartupScreen()
+            analytics.userLogout(force: false)
+        } catch let error {
+            if error.isInternetError {
+                errorMessage = CoreLocalization.Error.slowOrNoInternetConnection
+            } else {
+                errorMessage = CoreLocalization.Error.unknownError
+            }
+        }
```

Comment:
It's better to keep that logic; it's done on purpose.
We must ignore server response for logout for offline and deprecated version cases.

Reply:
I'm not sure how this gets changed. I'd only change the method. Reverted the code.

Reply:
Got it. Maybe some merge issue. idk